### ISSUE TITLE
Adding SOVABIDS to the NeuroDesk Conda environment container

### ISSUE THREAD

**Issue by civier** [NeuroDesk/neurocontainers#91]
Hi Angela,

Can SOVABIDS be included in the container with the Python conda environment if it needs to be cloned from Github and then piped (https://sovabids.readthedocs.io/en/latest/README.html#installation), or do you only include things that can be conda-installed or piped directly from repositories? Is there a specific problem to clone things from Github first?

If we can add the SOVABIDS installation from Github, where should we do it? @  @yjmantilla Steff, any of you want to try it out?

NOTE: there is already an old version of SOVABIDS in the bidstools container, but since SOVABIDS also has a python interface (https://sovabids.readthedocs.io/en/latest/autoapi/index.html), some would like to use it from VSCode (or other tools available inside the Conda environment container), I guess

**Comment by stebo85** [comment]
This is possible, but we first need to finish #66 before we can do this

**Comment by yjmantilla** [comment]
I'm hella busy but maybe ring me when #66 is done.

**Comment by stebo85** [comment]
#66 is now completed.

If anyone is keen to add Sovabids you can follow this example:
https://github.com/NeuroDesk/neurocontainers/tree/master/recipes/mne

Cheers
Steffen

**Comment by yjmantilla** [comment]
@stebo85 I'll keep that in mind although my time has been really limited

**Comment by civier** [comment]
Hi Steff (
@neuro-sevas)
I tag you on this issue, in case you want to give a SOVABIDS container a go. If you use the mne example, you can just keep VScode in, as people might like to interact with SOVABIDS through python code.
Best,
Oren

**Comment by civier** [comment]
Right now the urgency is to have the newest version of BIDScoin, so I will work on SOVABIDS only later. Marked as "wontfix right now".

**Comment by yjmantilla** [comment]
From Oren_

If one of you finds time to build a SOVABIDS container for Neurodesk, I just wanted to let you know that I already built a BIDScoin container with the latest version:
https://github.com/NeuroDesk/neurocontainers/tree/master/recipes/bidscoin
So you can use it as a template for the SOVABIDS container.

**Comment by civier** [comment]
SOVABIDS is a special case because it has GUI that can be used independently, but it can also be used via Python. So I'm not sure what will be the best solution for it:
One option is to include it only in the Python environment, but add a menu entry for the GUI (but then people need to download the whole environment, in case they don't work with CVMFS)
Another option is to include SOVABIDS both in the Python environment and in a separate container.
Third option is to only have SOVABIDS in a separate container -- that might be easier for the developers because the python environment is complicated, and it might be too much overhead to make it build properly everytime we want to only upgrade SOVABIDS (which will be frequent, given that SOVABIDS is still a young software)
Any thoughts?

**Comment by yjmantilla** [comment]
I think the third option (separate container) seems to be the most healthy one overall.

**Comment by stebo85** [comment]
closing because it's completed
